libs/payments/cart/src/lib/factories.ts

```diff
@@ -0,0 +1,56 @@
+import { faker } from '@faker-js/faker';
+import { Invoice, Subscription, TaxAmount, CartState } from './types';
+import { TaxAddress, Cart } from '../gql';
+
+export const CartFactory = (override?: Partial<Cart>): Cart => ({
+  id: faker.string.uuid(),
+  state: CartState.START,
+  offeringConfigId: faker.helpers.arrayElement([
+    'vpn',
+    'relay-phone',
+    'relay-email',
+    'hubs',
+    'mdnplus',
+  ]),
+  interval: 'monthly',
+  nextInvoice: InvoiceFactory(),
+  createdAt: new Date(faker.date.recent()).getTime(),
+  updatedAt: new Date(faker.date.recent()).getTime(),
+  amount: parseInt(faker.finance.amount()),
```

Comment:
faker.number.int() would be a better fit for when we want non-decimal representation of amounts.